Source/Core/Core/Host.h

```diff
@@ -31,6 +31,7 @@ void Host_NotifyMapLoaded();
 void Host_RefreshDSPDebuggerWindow();
 void Host_RequestRenderWindowSize(int width, int height);
 void Host_UpdateDisasmDialog();
+void Host_UpdateBreakpoints();
```

Comment:
Please don't add anything new to the legacy Host interface. Instead, I'd suggest letting the UIs set a callback.